Assets/HoloToolkit/Utilities/Editor/AutoConfigureMenu.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+using UnityEditor;
+using UnityEngine;
+
+namespace HoloToolkit.Unity
+{
+    /// <summary>
+    /// Configuration options derived from here: 
+    /// https://developer.microsoft.com/en-us/windows/holographic/unity_development_overview#Configuring_a_Unity_project_for_HoloLens
+    /// </summary>
+
+    
+    public class AutoConfigureMenu : MonoBehaviour
+    {
+
+        [MenuItem("HoloToolkit/Configure/Show Help", false, 3)]
+        public static void ShowHelp()
+        {
+            Application.OpenURL("https://developer.microsoft.com/en-us/windows/holographic/unity_development_overview#Configuring_a_Unity_project_for_HoloLens");
```

Comment:
Hmm I was wondering why should HoloToolkit > Help menu point to Unity development overview.
Perhaps to GettingStarted.md or a new page for help?